x-pack/plugins/screenshotting/server/browsers/chromium/driver.ts

```diff
@@ -72,18 +74,13 @@ interface InterceptedRequest {
 
 const WAIT_FOR_DELAY_MS: number = 100;
 
-function getDisallowedOutgoingUrlError(interceptedUrl: string) {
-  return new Error(
-    `Received disallowed outgoing URL: "${interceptedUrl}". Failing the request and closing the browser.`
-  );
-}
-
 /**
  * @internal
  */
 export class HeadlessChromiumDriver {
   private listenersAttached = false;
   private interceptedCount = 0;
+  private screenshottingError$ = new Subject<Error>();
```

Comment:
It's better to encapsulate the subject side.

```suggestion
  private screenshottingErrorSubject = new Subject<Error>();
  readonly screenshottingError$ = this.screenshottingErrorSubject.asObservable();
```

And if we use the `readonly` modifier, there will be no need in the getter below as it's equal in the TypeScript code.